Projects/Dotmim.Sync.MySql/Builders/MySqlDbMetadata.cs

```diff
@@ -43,22 +45,26 @@ public override int GetMaxLengthFromOwnerDbType(object dbType, int maxLength)
                 string lowerType = typeName.ToLowerInvariant();
                 switch (lowerType)
                 {
-                    case "MEDIUMTEXT":
-                    case "LONGTEXT":
-                    case "TINYBLOB":
-                    case "MEDIUMBLOB":
-                    case "LONGBLOB":
-                    case "BLOB":
-                    case "JSON":
-                    case "TINYTEXT":
+                    case "mediumtext":
+                    case "longtext":
+                    case "tinyblob":
+                    case "mediumblob":
+                    case "longblob":
+                    case "blob":
+                    case "json":
+                    case "tinytext":
                         return 0;
-                    case "TEXT":
-                    case "NCHAR":
-                    case "NVARCHAR":
-                    case "VARCHAR":
-                    case "CHAR":
-                    case "ENUM":
-                    case "SET":
+                    case "varbinary":
+                        return 65535;
```

Comment:
I would have done a little different here, to handle the column maxlength:
(macro code)
if maxLength > 0 return Math.Min(maxLength, 65535) else return 65535;